.gitignore

```diff
@@ -1,12 +1,14 @@
+# IDE
 .idea/
 .vscode/
 
+# Environement
 .env
 
+# Dependencies
 node_modules/
 /.pnp
 .pnp.js
 
-*.js
-
+# Build
 dist/
```

Comment:
Please consider using [the full NextJS gitignore file](https://github.com/akiran/nextjs-demo/blob/master/.gitignore)